src/components/verification-grid-settings/verification-grid-settings.ts

```diff
@@ -48,7 +52,63 @@ export class VerificationGridSettingsComponent extends AbstractComponent(LitElem
     this.isFullscreen = document.fullscreenElement === this.verificationGrid;
   }
 
-  public render() {
+  private handleGridSizeChange(event: ChangeEvent<HTMLInputElement>) {
+    if (!this.verificationGrid) {
+      throw new Error("Could not find associated verification grid component");
+    }
+
+    // we use the Number constructor here instead of parseInt so that if
+    // the value of the input is not a number the number constructor will return
+    // NaN and we can hard fail and not change the grid size instead of
+    // changing to a grid size that the user did not expect
+    const inputValue = event.target.value;
+    const newGridSize = Number(inputValue);
+
+    // we use isFinite here to check that the value is not NaN, and that
+    // values such as Infinity are not considered as a valid grid size
+    if (!isFinite(newGridSize)) {
+      throw new Error("New grid size value could not be converted to a number");
+    }
+
+    this.verificationGrid.gridSize = newGridSize;
+  }
+
+  private gridSizeSettingsTemplate() {
+    if (!this.verificationGrid) {
+      throw new Error("Could not find associated verification grid component");
+    }
+
+    const currentGridSize = this.verificationGrid.gridSize;
+
+    return html`
+      <sl-dropdown>
+        <sl-tooltip slot="trigger" content="Change the verification grid size">
+          <button
+            id="grid-size-trigger"
+            class="oe-btn-secondary"
+            ?disabled="${!this.verificationGrid}"
+            aria-label="Change the verification grid size"
+          >
+            <sl-icon name="grid" class="large-icon"></sl-icon>
+          </button>
+        </sl-tooltip>
+
+        <label>
+          ${currentGridSize}
+          <input
+            id="grid-size-input"
+            @change="${this.handleGridSizeChange}"
+            type="range"
+            value="${currentGridSize}"
+            min="1"
+            max="12"
+          />
+        </label>
```

Comment:
This looks a lot more consistent. I like it

I will also be changing the media-controls brightness/contrast so that the look is consistent across components